src/services/syncService.js

```
@@ -0,0 +1,147 @@
+
+    // Create a document in Firebase with all custom tiles
+    await setDoc(
+      doc(db, 'user-data', user.uid),
+      {
+        customTiles,
+        lastUpdated: new Date(),
+      },
+      { merge: true }
+    );
+
+    console.log('Custom tiles synced: ' + customTiles.length);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)